transports/upgrades/amazonsqs-6to7.md

```diff
@@ -9,8 +9,38 @@ upgradeGuideCoreVersions:
  - 9
 ---
 
+## Configuration options
+
+The SQS transport configuration options that have not changed have been moved to the `SqsTransport` class. See the following table for further information:
+
+| Version 6 configuration option | Version 7 configuration option |
+| --- | --- |
+| EnableV1CompatibilityMode | EnableV1CompatibilityMode |
+| MapEvent | MapEvent |
+| MaxTimeToLive | MaxTimeToLive |
+| Policies | Policies |
+| TopicNameGenerator | TopicNameGenerator |
+| TopicNamePrefix | TopicNamePrefix |
+| QueueNamePrefix | QueueNamePrefix |
+| Policies.AddAccountCondition | Policies.AccountCondition |
+| Policies.AddTopicNamePrefixCondition | Policies.TopicNamePrefixCondition |
+| Policies.AddNamespaceCondition | Policies.TopicNamespaceConditions |
+| Policies.AssumePolicyHasAppropriatePermissions | Policies.SetupTopicPoliciesWhenSubscribing |
+
 ## V1 Compatibility Mode
 
 The option to serialize the `TimeToBeReceived` and `ReplyToAddress` message headers in the message envelope for compatibility with endpoints using version 1 of the transport is no longer available.
 
 Make sure that all V1 endpoints are upgraded to a supported version and remove the `transport.EnableV1CompatibilityMode();` configuration option.
+
+## S3 configuration
```

Comment:
Not sure I'm following on what these were moved for. Reading the text here, nothing is describing what has even changed. There's something missing from this text to make it an "upgrade guide".

Reply:
The v5 way was to configure S3 like this

```
var transport = endpointConfiguration.UseTransport<SqsTransport>();
var s3Configuration = transport.S3(
    bucketForLargeMessages: "nsb-sqs-messages",
    keyPrefix: "my/sample/path");
```

https://docs.particular.net/transports/sqs/configuration-options?version=sqs_5.7#offload-large-messages-to-s3

The v6 way of doing it is:

```
var transport = new SqsTransport
{
    S3 = new S3Settings(
        bucketForLargeMessages: "nsb-sqs-messages",
        keyPrefix: "my/sample/path")
};
```

so I think this should be moved back to the 5to6upgrade guide?

Reply:
Moved back and reworded to read better